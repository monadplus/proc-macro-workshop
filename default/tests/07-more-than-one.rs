#[derive(Debug, PartialEq, Eq, Default)]
struct A {
    a: u8,
    b: String,
}

#[derive(Debug, PartialEq, Eq, Default)]
struct B {
    a: u8
}

#[derive(Debug, PartialEq, Eq, derive_default::Default)]
enum Letters {
    #[default]
    A(A, u8),
    #[default]
    B(B)
}

fn main() {}
