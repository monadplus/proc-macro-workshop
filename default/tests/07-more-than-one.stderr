error: #[default] is defined more than once
  --> tests/07-more-than-one.rs:16:5
   |
16 | /     #[default]
17 | |     B(B)
   | |________^
